modules/backend/widgets/Lists.php

```diff
@@ -1404,6 +1402,17 @@ public function addFilter(callable $filter)
     // Searching
     //
 
+    /**
+     * Applies a search term to the list results, same as setSearchTerm method does
+     * but it will reset pagination to page 1 after new submited search term.
+     * @param string $term
+     */
+    public function submitSearchTerm($term)
```

Comment:
I would rather that `setSearchTerm($term)` becomes `setSearchTerm($term, $resetPagination = false)` instead of adding an entirely new method that does basically the same thing.

Reply:
Yep, thats better idea!